modules/Microsoft.DigitalTwins/digitalTwinsInstances/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,166 @@
+resource eventHubNamespace 'Microsoft.EventHub/namespaces@2022-10-01-preview' = {
+    name: eventHubNamespaceName
+    location: location
+    properties: {
+        zoneRedundant: false
+        isAutoInflateEnabled: false
+        maximumThroughputUnits: 0
+    }
+}
+
+resource eventHub 'Microsoft.EventHub/namespaces/eventhubs@2022-10-01-preview' = {
+    name: eventHubName
+    parent: eventHubNamespace
+}
+
+resource serviceBus 'Microsoft.ServiceBus/namespaces@2022-10-01-preview' = {
+    name: serviceBusName
+    location: location
+    properties: {
+        zoneRedundant: false
+    }
+}
+
+resource serviceBusTopic 'Microsoft.ServiceBus/namespaces/topics@2022-10-01-preview' = {
+    name: serviceBusTopicName
+    parent: serviceBus
+}
+
+resource eventGridDomain 'Microsoft.EventGrid/domains@2022-06-15' = {
+    name: eventGridDomainName
+    location: location
+    properties: {
+        disableLocalAuth: false
+    }
+}
+
+resource eventGridTopic 'Microsoft.EventGrid/domains/topics@2022-06-15' = {
+    name: eventGridTopicName
+    parent: eventGridDomain
+}
+
+resource evhrbacAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+    name: guid(managedIdentity.id, 'evhrbacAssignment')
+    scope: eventHubNamespace
+    properties: {
+        roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '2b629674-e913-4c01-ae53-ef4638d8f975')
+        principalId: managedIdentity.properties.principalId
+        principalType: 'ServicePrincipal'
+    }
+}
+
+resource sbhrbacAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+    name: guid(managedIdentity.id, 'sbrbacAssignment')
+    scope: serviceBus
+    properties: {
+        roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '69a216fc-b8fb-44d8-bc22-1f3c2cd27a39')
+        principalId: managedIdentity.properties.principalId
+        principalType: 'ServicePrincipal'
+    }
+}
+
+@description('The resource ID of the created Virtual Network Subnet.')
+output subnetResourceId string = virtualNetwork.properties.subnets[0].id
+
+@description('The principal ID of the created Managed Identity.')
+output managedIdentityPrincipalId string = managedIdentity.properties.principalId
+
+@description('The resource ID of the created Private DNS Zone.')
+output privateDNSResourceId string = privateDNSZone.id
+
+output eventhubNamespaceName string = eventHubNamespace.name
+
+output eventHubId string = eventHub.id
+
+output eventhubName string = eventHub.name
+
+output serviceBusName string = serviceBus.name
+
+output serviceBusTopicName string = serviceBusTopic.name
+
+output eventGridDomainName string = eventGridDomain.name
+
+output eventGridTopicName string = eventGridTopic.name
+
+output eventGridEndpoint string = eventGridDomain.properties.endpoint
+
+output eventGridTopicId string = eventGridTopic.id
+
+output eventGridDomainId string = eventGridDomain.id
+
+output managedIdentityId string = managedIdentity.id
```

Comment:
Each time you use `id` please use `resourceId` instead and add the output metadata. 
It doesn't provide a functional advantage, but makes it easier to understand what one is looking at and leads by example :)